tools/update_tool

```diff
@@ -0,0 +1,466 @@
+            continue
+
+        prev_manifest, prev_imi = extract(prev, do_filter=not args.no_filter)
+        curr_manifest, curr_imi = extract(curr, do_filter=not args.no_filter)
+
+        manifest_status = compare(prev_manifest, curr_manifest)
+        image_info_status = compare(prev_imi, curr_imi)
+
+        # Since changes are filtered because we encounter UUID volatility on
+        # some entries, do not print the ones that after this filtering have in
+        # fact no changes at all.
+        if manifest_status == "unchanged" and image_info_status == "unchanged":
+            continue
+
+        print(f"{BOLD}{filename}{RESET}:")
+
+        if manifest_status != "unchanged":
+            diff("manifest", prev_manifest, curr_manifest, tool=args.manifest_diff_tool)
+
+        if image_info_status != "unchanged":
+            diff("image-info", prev_imi, curr_imi, tool=args.image_info_diff_tool)
+
+
+def report_command(args):
+    """
+    Prints a summary of the DB update as an ascii table. Or create a github TODO
+    list to be included in the update PR
+    """
+    print_header(args.github_markdown)
+    for file in list_changed_files():
+        filename = file.split("/")[-1].split(".")[0]
+
+        prev = get_db_entry_at_previous_version(file)
+        curr = get_db_entry_at_current_version(file)
+
+        if not (prev or curr):
+            continue
+
+        prev_manifest, prev_imi = extract(prev, do_filter=not args.no_filter)
+        curr_manifest, curr_imi = extract(curr, do_filter=not args.no_filter)
+
+        manifest_status = compare(prev_manifest, curr_manifest)
+        image_info_status = compare(prev_imi, curr_imi)
+
+        # Since changes are filtered because we encounter UUID volatility on
+        # some entries, do not print the ones that after this filtering have in
+        # fact no changes at all.
+        if manifest_status == "unchanged" and image_info_status == "unchanged":
+            continue
+
+        start_row(filename, args.github_markdown)
+
+        print_status(manifest_status, args.github_markdown)
+        print_status(image_info_status, args.github_markdown, last=True)
+
+        end_row(args.github_markdown)
+
+
+def main():
+    parser = argparse.ArgumentParser(description="""A tool to explore a DB
+    update commit.""")
+
+    parser.add_argument(
+        "--no-filter",
+        action="store_true",
+        default=False,
+        help="Disable UUID masking"
+    )
+    subparsers = parser.add_subparsers(
+        title="command",
+        required=True,
+        dest='command',
+        help='Command to execute')
+    parser_report = subparsers.add_parser('report')
+
+    # Report command
+    parser_report.add_argument(
+        "--github-markdown",
+        action="store_true",
+        default=False,
+        help="Create a TODO list for github to ask a reviewer to check everything"
+    )
+
+    # Diff command
+    parser_diff = subparsers.add_parser('diff')
```

Comment:
I think we should have `description=` for all subparsers so it's easier to find out what the subcommands do,